datasets/s2orc/README.md

```diff
@@ -0,0 +1,224 @@
+
+### Data Splits
+
+There is no train/dev/test split given in the dataset
+
+## Dataset Creation
+
+### Curation Rationale
+
+Academic papers are an increasingly important textual domain for natural language processing (NLP) research. Aside from capturing valuable knowledge from humankind’s collective research efforts, academic papers exhibit many interesting characteristics – thousands of words organized into sections, objects such as tables, figures and equations, frequent inline references to these objects, footnotes, other papers, and more
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+To construct S2ORC, we must overcome challenges in (i) paper metadata aggregation, (ii) identifying open access publications, and (iii) clustering papers, in addition to identifying, extracting, and cleaning the full text and bibliometric annota- tions associated with each paper. The pipeline for creating S2ORC is:
+
+1) Process PDFs and LATEX sources to derive metadata, clean full text, inline citations and references, and bibliography entries,
+2) Select the best metadata and full text parses for each paper cluster,
+3) Filter paper clusters with insufficient meta- data or content, and
+4) Resolve bibliography links between paper clusters in the corpus.
+
+#### Who are the source language producers?
+
+S2ORC is constructed using data from the Se- mantic Scholar literature corpus (Ammar et al., 2018). Papers in Semantic Scholar are derived from numerous sources: obtained directly from publishers, from resources such as MAG, from various archives such as arXiv or PubMed, or crawled from the open Internet. Semantic Scholar clusters these papers based on title similarity and DOI overlap, resulting in an initial set of approxi- mately 200M paper clusters.
+
+### Annotations
+
+#### Annotation process
+
+[More Information Needed]
+
+#### Who are the annotators?
+
+[More Information Needed]
+
+### Personal and Sensitive Information
+
+[More Information Needed]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+[More Information Needed]
+
+### Discussion of Biases
+
+[More Information Needed]
+
+### Other Known Limitations
+
+[More Information Needed]
+
+## Additional Information
+
+### Dataset Curators
+
+List the people involved in collecting the dataset and their affiliation(s). If funding information is known, include it here.
+
+### Licensing Information
+
+Provide the license and link to the license webpage if available.
+
+### Citation Information
+
+Provide the [BibTex](http://www.bibtex.org/)-formatted reference for the dataset. For example:
+```
+@article{article_id,
+  author    = {Author List},
+  title     = {Dataset Paper Title},
+  journal   = {Publication Venue},
+  year      = {2525}
+}
+```
+
+If the dataset has a [DOI](https://www.doi.org/), please provide it here.
```

Comment:
Can you provide the missing information here ? Or use `[More Infirmation Needed]`